docs/intro/installation.md

```diff
@@ -0,0 +1,359 @@
+[Agave Github repo](https://github.com/anza-xyz/agave/releases).
+
+<Callout title="Updating the Solana CLI">
+
+In the future, you can use the Solana CLI to update itself based on which latest
+version is available: `agave-install update`
+
+Agave is the validator client from [Anza](https://www.anza.xyz/), formerly known
+as Solana Labs validator client.
+
+</Callout>
+
+### Install Anchor CLI
+
+[Anchor](https://www.anchor-lang.com/) is a framework for developing Solana
+programs. The Anchor framework leverages Rust macros to simplify the process of
+writing Solana programs.
+
+To install and manage anchor versions, use `avm`, the anchor version manager.
+Since `avm` is installed via `cargo` (the Rust package manager), the
+installation steps will be the same for all the operating systems.
+
+We can then use `avm` to install the desired version of the Anchor framework.
+
+#### Install avm
+
+```shell
+cargo install --git https://github.com/coral-xyz/anchor avm --locked --force
+```
+
+#### Install Anchor using avm
+
+To install the `latest` version of anchor using `avm`:
+
+```shell
+avm install latest
+avm use latest
+```
+
+After the anchor installation is complete, you can verify anchor was installed
+by checking the installed version:
+
+```shell
+anchor --version
+```
+
+> If you do not see an output or receive an error, you may need to restart your
+> terminal. If you are still running into errors, you can also install the
+> latest version directly without locking in the dependencies:
+>
+> ```shell
+> cargo install --git https://github.com/coral-xyz/anchor --tag v0.30.1 anchor-cli --force
+> ```
+
+### Create a file system wallet
+
+To deploy a program with the Solana CLI, you will need a Solana wallet with SOL
+tokens to pay for the cost of transactions and data storage on the blockchain.
+
+To create a file system wallet to use during local development, run the
+following command:
+
+```shell
+solana-keygen new
+```
+
+By default, the `solana-keygen` command will create a new file system wallet
+located at `~/.config/solana/id.json`. You can manually specify the output file
+location using the `--outfile /path` option.
+
+<Callout type="note">
+
+If you already have a file system wallet saved at the default location, this
+command will **NOT** override it unless you explicitly force override using the
+`--force` flag.
+
+</Callout>
+
+With your new file system wallet created, you must tell the Solana CLI to use
+this wallet to deploy and take ownership of your on-chain program:
+
+```shell
+solana config set -k ~/.config/solana/id.json
+```
+
+You can view your current Solana CLI configuration settings with the following
+command:
+
+```shell
+solana config get
+```
+
+### Airdrop SOL tokens to your wallet
+
+Once you've set up your local wallet, request an airdrop of SOL to fund your
+wallet. You need SOL to pay for transaction fees and to deploy programs.
+
+Set your cluster to the devnet:
+
+```shell
+solana config set -ud
```

Comment:
Should we start with devnet in a local installation guide? Seems contrary to what we want to achieve in the guide.

Reply:
When running solana-test-validator, you generally don't need to airdrop since the default keypair starts with 500000000 SOL.